### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Alesli**

### CODE REVIEW

File: post-service/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# DataSource properties
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.datasource.url=jdbc:postgresql://posts-db-container:5432/postsdb
+spring.datasource.username=root
+spring.datasource.password=password
```
Comment (jvma0) [inline post-service/src/main/resources/application.properties]: Across the entire application port, username and password should be taken from the environment variables

File: user-service/src/main/java/com/training/docker/entity/User.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
```
Comment (jvma0) [inline user-service/src/main/java/com/training/docker/entity/User.java]: Not sure that you need this

File: user-service/src/main/java/com/training/docker/service/impl/UserServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    @Override
+    public boolean deleteById(Long id) {
+        var existingUser = repository.findById(id);
+        if (existingUser.isPresent()) {
+            repository.deleteById(id);
+            return true;
+        } else {
+            return false;
+        }
+    }
```
Comment (jvma0) [inline user-service/src/main/java/com/training/docker/service/impl/UserServiceImpl.java]: Better to throw some kind of not found exception/return 404 in the case when the user is not found by id.

**Review by jvma0 — COMMENTED** [review]
Reorganize your docker files, just put a single docker file for a particular service for each service folder + use the same port for your services, for example, 8080 for services and 5432 for postgres, and map to another port within the docker-compose

File: post-service/src/main/resources/application.properties (hunk 1 above)
Comment (Alesli) [reply post-service/src/main/resources/application.properties]: spring.datasource.url=${DB_URL}
spring.datasource.username=${DB_USERNAME}
spring.datasource.password=${DB_PASSWORD}

File: user-service/src/main/java/com/training/docker/service/impl/UserServiceImpl.java (hunk 3 above)
Comment (Alesli) [reply user-service/src/main/java/com/training/docker/service/impl/UserServiceImpl.java]: } else {
            throw new UserNotFoundException(id);
        }

**Comment by jvma0** [comment]
Good

**Pull request merged**